app-backend/api/src/test/scala/organization/OrganizationSpec.scala

```diff
@@ -46,19 +46,19 @@ class OrganizationSpec extends WordSpec
         responseAs[PaginatedResponse[Organization]]
       }
     }
-    "require authorization for creation of new organizations" in {
-      val newOrg = Organization.Create("Test Organization")
-      Post("/api/organizations")
-        .withHeadersAndEntity(
-        List(authHeader),
-        HttpEntity(
-          ContentTypes.`application/json`,
-          newOrg.asJson.noSpaces
-        )
-      ) ~> baseRoutes ~> check {
-        rejection
-      }
-    }
+//     "require authorization for creation of new organizations" in {
```

Comment:
I'd rather this be removed entirely with a comment about what should go there eventually (not sure if an issue is appropriate, you be the judge I guess) -- it's not obvious from a quick read why it's commented out, and future me is going to be very confused

Reply:
Fair point! Will do.